### PR TITLE
Add NoBorder Gray theme

### PR DESCRIPTION
**Pull request by MystieHum**
hello! i made another theme, this time a slightly different variant of NoBorder Dark

### CODE REVIEW

**Comment by MystieHum** [comment]
@laymonage

**Review by laymonage — CHANGES_REQUESTED** [review]
Thanks, but there isn't enough contrast for links. Could you please fix it?

image

image

**Comment by MystieHum** [comment]
@laymonage i think that's enough contrast now

File: styles/themes/noborder_gray.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  --color-fg-default: rgb(235 235 245 / 86%);
+  --color-fg-muted: rgb(235 235 245 / 60%);
+  --color-fg-subtle: rgb(235 235 245 / 50%);
+  --color-canvas-default: rgb(30 30 32 / 100%);
+  --color-canvas-overlay: rgb(30 30 32 / 100%);
+  --color-canvas-inset: rgba(var(--bg-default), 0.85);
+  --color-canvas-subtle: rgba(var(--bg-default), 1);
+  --color-border-default: rgba(var(--bg-default), 0.85);
+  --color-border-muted: rgb(175 184 193 / 20%);
+  --color-neutral-muted: rgb(175 184 193 / 20%);
```
Comment (laymonage) [inline styles/themes/noborder_gray.css:69]: I changed these into `19%` for a better contrast ratio, hope that's OK!

**Review by laymonage — APPROVED** [review]

File: styles/themes/noborder_gray.css (hunk 1 above)
Comment (MystieHum) [reply styles/themes/noborder_gray.css]: that's ok!

**Pull request merged**